build.sbt

```diff
@@ -11,36 +13,38 @@ lazy val core = project
   .settings(commonSettings)
   .settings(
     name := "sbt-http4s-org",
-    addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.4.6"),
-    addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.9.4"),
+    addSbtPlugin("com.codecommit" % "sbt-spiewak" % "0.17.0"),
     addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.15"),
     addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.6.0"),
```

Comment:
The Java formatter plugin is only used in blaze right now, and we could move it back there.